### DISCUSSION TITLE
allow user define the git exe location

### DISCUSSION THREAD

**Discussion by XiaochuanWang** [extrawurst/gitui#1009] [Q&A] [answered]
hi

I'm trying to use this app on the PC in my office, and the git is not in the system PATH. The company IT doesn't allow me to modify the system PATH due to security reasons. I'm just wondering if you can add a feature that allows users to define the git exe location?

e.g. 

gitui.exe --git-loc "D:\softwares\git.exe"

Cheers

**Comment by extrawurst** [comment, marked as answer]
Gitui does not use the git scripts, why do you think you need this?
